dashboard/app/views/pd/application/teacher1920_application_mailer/_partner_signature.html.haml

```diff
@@ -8,7 +8,7 @@
   - else
     = link_to @application.regional_partner.name, CDO.studio_url('/pd/regional_partner_contact/new', CDO.default_scheme)
 - else
-  Anthonette Peña
+  Liz Gauthier
```

Comment:
This change to a signature partial changes the signature on many emails:

- `accepted_no_cost_registration__without_partner`
- `confirmation__without_partner`
- `declined__without_partner`
- `principal_approval__without_partner`
- `principal_approval_completed__without_partner`
- `principal_approval_teacher_reminder__without_partner`
- `registration_sent__without_partner`
- `waitlisted__without_partner`